low-level/sensors/impl/generic/raid_integrity_data.py

```diff
@@ -210,7 +230,7 @@ def _get_devices(self):
 
```

Comment:
Adding comment here since I could not add at line #224 above.
Looks like the current code will send multiple MISSING alerts if the device_array is empty for every run. The ask was to limit the number of alerts.

Reply:
This scenario is covered in RAIDSensor.
And In RAIDIntegritySensor, The missing alert will be send once per run.

Reply:
Lets remove this alert from here then.